lib/src/widgets/typing/ds_dot.widget.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
```

Comment:
Could move this widget into widgets/chat folder

Reply:
Done